pkg/config/data/config_schema_v1.0.json

```diff
@@ -0,0 +1,252 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
+  "description": "The root schema comprises the entire JSON document.",
+  "default": {},
+  "examples": [
+    {
+      "build": {
+        "cuda": "10.1",
+        "gpu": true,
+        "python_version": "3.8",
+        "python_packages": [
+          "pytorch==1.4.0"
+        ],
+        "pre_install": [
+          "cowsay moo"
+        ],
+        "python_requirements": "my-requirements.txt",
+        "system_packages": [
+          "ffmpeg",
+          "libavcodec-dev"
+        ],
+        "run": [
+          "curl -L https://github.com/cowsay-org/cowsay/archive/refs/tags/v3.7.0.tar.gz | tar -xzf -",
+          "cd cowsay-3.7.0 && make install"
+        ]
+      },
+      "image": "registry.hooli.corp/jazz-solo-model",
+      "predict": "predict.py:JazzSoloComposerPredictor"
+    }
+  ],
```

Comment:
It'd be great to remove all the superfluous stuff here. Anything that's not actually needed to validate the config.

Reply:
These are just example strings that were generated automatically in the json schema. This is actually useful for users if they want to see examples of how certain variables need to be declared.
This was directly taken from the cog.md documentation.